docs/functions.md

```diff
@@ -46,7 +46,7 @@ export default async function (event) {
 |payload|string|The payload/body of the message you want to send.
 |headers|object&lt;string,string&gt;|The headers/metadata of the message you want to send.
 |channel|string|The channel/topic you want to send the message to. Defaults to `event.channel`, i.e., the same channel as the received event.
-|server|string|The server/broker you want to send the message to. Defaults to `event.serverName`, i.e., the same server as the received event.
+|server|string|The server/broker you want to send the message to. Must be present when function returns a `send` event.
```

Comment:
Why can't we default it to the same event as the one used to receive the event? 🤔

Reply:
> Why can't we default it to the same event as the one used to receive the event? 🤔

We are doing this in the case of the Reply event, according to docs both reply and send events have the same type which is an `array<OutboundMessage>`. In the Reply event, the message is sent to the same channel that it came from. but send allows you to send it to 
- the same channel 
- and some other channel or channels (all channels in a server)

maybe we do not need a reply event, we can just return an array of events that could have a server property or not, if it does not have one we reply and if it does we send it to that specific server.

Reply:
But server != connection. `reply` uses the same connection to reply. For instance, in the case of WS or HTTP, it replies to the specific client who made the "request", not all the clients connected to that server, which is what `send` would do.

In any case, what we're changing here is that we're removing the default value. I honestly don't see a reason to do it. For instance, I receive a message from an MQTT broker and, as a result, I want to send another one to the broker. I'd use `send` there because even though `reply` would do the same it might give the impression that it's going to respond to the message, which is not really possible on MQTT.